src/app/(private)/roles/[id]/page.tsx

```diff
@@ -347,6 +395,24 @@ const Page: NextPage<{ params: { slug: string; id: string } }> = ({
               <h1 className="text-2xl font-bold">{t('role.detailsTitle')}</h1>
               {roleDetail.status !== 'DELETED' && (
                 <div className="flex items-center gap-4">
+                  {userPermissions.includes(Permission.ROLE_UPDATE) &&
+                  roleDetail.status === 'ACTIVE' ? (
+                    <Button
+                      type="button"
+                      variant="outline"
```

Comment:
type="button" a gerek yok

Reply:
diğer türlü submit gibi çalışıyor ve event.preventDefault() eklemek gerekir. ayrıca diğer buttonlarda aynı şekilde type belirtilmiş aykırılık etmedim :)

Reply:
form içinde değilken submit gibi çalışmaz diye düşünmüştüm. zaten diğer butonlardan da silelim gerek yoksa 😄 burda gözüme çarptı diye söyledim. ayrıca çıkarılırsa veya eklenirse daha iyi olacağını düşündüğün herşey için lütfen aykırılık et 👍

Reply:
type="button" hali ile düzgün çalışıyor submit etmemesi içinde doğru olduğunu düşünüyorum.